src/include/duckdb/storage/storage_manager.hpp

```diff
@@ -87,6 +94,9 @@ class StorageManager {
 	//! When loading a database, we do not yet set the wal-field. Therefore, GetWriteAheadLog must
 	//! return nullptr when loading a database
 	bool load_complete = false;
+	//! The allocation size of blocks managed by this storage manager. Defaults to DEFAULT_BLOCK_ALLOC_SIZE.
+	//! This is NOT the actual memory available on a block (block_size).
+	const idx_t block_alloc_size;
```

Comment:
Can the block alloc size also be removed from the storage manager and stay only in the block manager?